rest/model/project_event.go

```diff
@@ -144,24 +150,34 @@ func (a *APIProjectAlias) ToService() (interface{}, error) {
 func (a *APIProjectAlias) BuildFromService(h interface{}) error {
 	switch v := h.(type) {
 	case *model.ProjectAlias:
-		APITags := []APIString{}
-		for _, tag := range v.Tags {
-			APITags = append(APITags, ToAPIString(tag))
+		APITaskTags := []APIString{}
+		APIVariantTags := []APIString{}
+		for _, tag := range v.TaskTags {
+			APITaskTags = append(APITaskTags, ToAPIString(tag))
+		}
+		for _, tag := range v.VariantTags {
+			APIVariantTags = append(APIVariantTags, ToAPIString(tag))
 		}
```

Comment:
should we have helper methods to convert []string to []APIString (and the inverse?)

Reply:
We maybe should :) This appears all over the codebase.